Casks/o/openhv.rb

```diff
@@ -0,0 +1,28 @@
+cask "openhv" do
+  version "20230917"
+  sha256 "c7f0c0260f690d4039cf7146faad01bf1731b8b77a28eeb0cb9573200c5bc95f"
+
+  url "https://github.com/OpenHV/OpenHV/releases/download/#{version}/OpenHV-#{version}.dmg",
+      verified: "github.com/OpenHV/OpenHV/"
+  name "OpenHV"
+  desc "Pixelart Science-Fiction Real-Time-Strategy game"
+  homepage "https://www.openhv.net/"
+
+  livecheck do
+    url :url
+    regex(/v?(\d+(?:\.\d+)*)/i)
+    strategy :github_latest
+  end
+
+  app "OpenHV.app"
+
+  zap trash: [
+    "~/Library/Application Support/OpenRA/hv-news.json",
+    "~/Library/Application Support/OpenRA/Logs/irc.log",
+    "~/Library/Application Support/OpenRA/maps/hv",
+    "~/Library/Application Support/OpenRA/ModMetadata/hv-*",
+    "~/Library/Application Support/OpenRA/Replays/hv",
+    "~/Library/Application Support/OpenRA/Saves/hv",
+    "~/Library/Application Support/OpenRA/Screenshots/hv",
```

Comment:
```suggestion
    "~/Library/Application Support/OpenRA/hv-news.json",
    "~/Library/Application Support/OpenRA/Logs",
    "~/Library/Application Support/OpenRA/maps",
    "~/Library/Application Support/OpenRA/ModMetadata",
    "~/Library/Saved Application State/net.openra.mod.hv.savedState",
```

It seems like replays, saves, and screenshots are user-created data, so we should keep them.